internals/cli/cmd_enter_test.go

```diff
@@ -242,3 +243,40 @@ func (s *PebbleSuite) TestEnterHelpCommandHelpArg(c *C) {
 	c.Check(stdout, Matches, "(?s).*\\bThe help command displays information about commands\\..*")
 	c.Check(exitCode, Equals, 0)
 }
+
+// TestEnterSubCommandWaits checks that the subcommand in enter
+// starts **after** the default services have started. It utilizes
+// the modification time of /proc/[PID] directory for checking.
+func (s *PebbleSuite) TestEnterSubCommandWaits(c *C) {
+	layerTemplate := dumbDedent(`
+		services:
+		  stat:
+		    override: replace
+		    command: /bin/sh -c 'stat -c "%y" /proc/$$ > $PEBBLE/stat; sleep 1;'
```

Comment:
I'm wondering if we can simplify, instead of using `stat` on `/proc/$$`, just output the date directly. `date --rfc-3339=ns > $PEBBLE/enter-wait`, for example. Similar with the exec command.

Reply:
This is better actually. Can't remember why I didn't try this before, it has been so long..